devtools/gradle-it-run/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-build-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-gradle-plugin-integration-test-run</artifactId>
+    <name>Quarkus - Gradle Plugin Run App IT</name>
+    <description>Quarkus - Gradle Plugin Run Application Integration Test</description>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+    </properties>
+
+    <dependencies>
```

Comment:
This does not have a dependency on quarkus-gradle-plugin-integration-test so its possible this can run before the jar it depends on is built.

Reply:
@stuartwdouglas that's a good point, I've added a dependency now - am curious if this helps to get this PR to finally pass on the Azure build server as well (and not just locally, as it always has)